bazel/repository_locations.bzl

```diff
@@ -708,11 +708,11 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Webassembly Micro Runtime",
         project_desc = "A standalone runtime with a small footprint for WebAssembly",
         project_url = "https://github.com/bytecodealliance/wasm-micro-runtime",
-        version = "a14a4487bb8b493bf6c68d83b03f12028d16f58a",
-        sha256 = "d68668e129f16a9ddd7a1a0da22b17905a25001ae2de398726d37880b61fee9e",
+        version = "15dd6515395236d1edbdda2de5740613c781ce3d",
+        sha256 = "534e789918952bb9c4c1cd5726c757daf736cdb3794a2e3cfa49b632d2b004a1",
```

Comment:
Could you update it to a more recent commit? It looks that there were some relevent bugfixes since then.

Reply:
Okay